.github/workflows/pages.yml

```
@@ -5,20 +5,27 @@ on:
   release:
     types: [published]
 
+concurrency:
+  group: ${{workflow}}-${{github.ref}}
```

actionlint (1.7.12)

[error] undefined variable "workflow". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)